superset/assets/src/explore/components/ExploreChartHeader.jsx

```diff
@@ -32,9 +41,22 @@ const propTypes = {
 };
 
 class ExploreChartHeader extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.fetchTags = fetchTags.bind(this, 'chart', props.chart.id, false);
```

Comment:
nit: you can use `autobind` here if you want 

https://github.com/andreypopp/autobind-decorator

Reply:
But here I'm binding more than just `this`, I'm actually building a partial function that can be passed to `<ObjectTags />`. From what I understand `autobind` will only bind `this`, no?

Reply:
Ahh your right I thought there was way for additional params